app/src/main/java/de/christinecoenen/code/zapp/app/main/MainActivity.kt

```diff
@@ -43,7 +44,10 @@ class MainActivity : AppCompatActivity(), MenuProvider {
 
 		setContentView(binding.root)
 
+		val startFragmentId = SettingsRepository(this).startFragment
```

Comment:
`settingsRepository ` has already been injected above:
```suggestion
		val startFragmentId = settingsRepository.startFragment
```